site/content/docs/5.3/examples/.stylelintrc

```diff
@@ -2,6 +2,8 @@
   "extends": [
     "stylelint-config-twbs-bootstrap"
```

Comment:
Might be better for maintenance if we inherited from the root .stylelintrc file. If anyone could push the change here, that'd help.